content/faq/tts/tts.md

```diff
@@ -77,4 +77,8 @@ If you’re having trouble initializing the RHVoice TTS engine on LineageOS or o
 
 - Now, you should be able to use RHVoice without any problems
 
-*In order to test the voice instructions, you can tap on "Test Voice Directions (TTS, Text-ToSpeech)" in OM "Settings → Voice Instruction" menu or you can actually start a navigation to receive any voice output, Organic Maps will not give you any voice instructions while you're standing still.*
+## Testing
+
+In order to test the voice instructions, you can tap on "Test Voice Directions (TTS, Text-ToSpeech)" in OM "Settings → Voice Instruction" menu or you can actually start a navigation to receive any voice output. Organic Maps will not give you any voice instructions while you're standing still.
+
+<img src='/images/screenshots/tts_test.png' width='400' alt='Screenshot TTS Test'>
```

Comment:
I don't think that encoding screenshot size in markdown is a good idea. It can/should be styled in css. You can add a markdown-style image instead.

Reply:
Sorry, but my knowledge about CSS styling is null, and I cannot find any example across the code as all the existing images follow this same size encoding type.

Can you please provide a CSS sample code that I could follow?

Reply:
We can leave it for future website redesign. I hope the hard-coded width will work for mobiles with a narrow screen.

Reply:
I've made some checks and the issue with narrow screens can be solved by replacing `width='400'` with `style='width:100%; max-width:400px'`